main.nf

```diff
@@ -301,6 +315,8 @@ if (params.skip_basecalling) {
         barcode_kit = params.barcode_kit ? "--barcode_kits $params.barcode_kit" : ""
         config = params.guppy_config ? "--config $params.guppy_config" : "--flowcell $params.flowcell --kit $params.kit"
         proc_options = params.guppy_gpu ? "--device $params.gpu_device --num_callers $task.cpus --cpu_threads_per_caller $params.guppy_cpu_threads --gpu_runners_per_device $params.guppy_gpu_runners" : "--num_callers 2 --cpu_threads_per_caller ${task.cpus/2}"
+        model_path = (local_model && params.guppy_model) ? "--model $local_model" : ""
+        if (!model_path) model_path = params.guppy_model ? "--model \$PWD/" : ""
```

Comment:
Why do we need `\$PWD/` here?

Reply:
Otherwise guppy will search the `/opt/ont/guppy/data/` directory for the file, not find it, and then throw an error.
![image](https://user-images.githubusercontent.com/37123979/69776161-1028bc00-11d6-11ea-8fac-369b65f3916d.png)

That's what happened in the above example - I passed a local file, which was staged properly but guppy looked within the guppy dir for it. With `\$PWD/` attached it will look in the current work dir where the file is staged